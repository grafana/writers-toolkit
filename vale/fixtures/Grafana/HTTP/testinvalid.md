Don't use:

- <http://grafana.com>
- [link text](http://grafana.com)
