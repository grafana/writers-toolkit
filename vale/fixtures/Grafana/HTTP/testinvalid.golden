testinvalid.md:3:4:Grafana.HTTP:Use 'https://grafana.com' instead of 'http://grafana.com'. The HTTP scheme is insecure and all grafana.com links must use HTTPS. 
testinvalid.md:4:15:Grafana.HTTP:Use 'https://grafana.com' instead of 'http://grafana.com'. The HTTP scheme is insecure and all grafana.com links must use HTTPS. 
